services/ui-src/src/components/layout/Header.test.tsx

```diff
@@ -16,44 +18,58 @@ describe("<Header />", () => {
       render(headerComponent);
     });
 
-    test("Navigation, Logo, Help and Menu is visible on Header", () => {
+    test("Logo, Help and Menu is visible on Header", () => {
       const header = screen.getByRole("navigation");
       expect(header).toBeVisible();
-      expect(screen.getByAltText("HCBS logo")).toBeVisible();
-      expect(screen.getByAltText("Help")).toBeVisible();
+      // find img elements
+      expect(screen.getByRole("img", { name: "HCBS logo" })).toHaveAttribute(
+        "alt",
+        "HCBS logo"
+      );
+      expect(screen.getByRole("img", { name: "Help" })).toHaveAttribute(
+        "alt",
+        "Help"
+      );
+      expect(screen.getByRole("img", { name: "Account" })).toHaveAttribute(
+        "alt",
+        "Account"
+      );
```

Comment:
Same suggestion here.

```suggestion
      // find img elements
      expect(screen.getByRole("img", { name: "HCBS logo" }))..toBeVisible()
      expect(screen.getByRole("img", { name: "Help" })).toBeVisible();
      expect(screen.getByRole("img", { name: "Account" })).toBeVisible();
```